src/pudl/cli.py

```diff
@@ -70,6 +73,10 @@ def main():
     coloredlogs.install(fmt=log_format, level='INFO', logger=logger)
 
     args = parse_command_line(sys.argv)
+    if args.logfile:
+        file_logger = logging.FileHandler(args.logfile)
+        file_logger.setFormatter(logging.Formatter(log_format))
+        logger.addHandler(file_logger)
```

Comment:
You're welcome :) I think that prefect may affect logging to some extent as well. Ideally you would be able to associate logs to specific tasks, but this may be only possible if you're running the flows via prefect server and that is not what we currently do. 

Having everything dumped to a file will help esp when we start running this in an automated fashion.